src/utils/constants.ts

```diff
@@ -257,3 +257,17 @@ export const detailsTabOptions: TabOptions = {
     },
   ],
 };
+
+export const pageTitles = {
+  home: 'NCEA Search Service Home',
+  date: 'NCEA Guided Search (Study Period)',
+  geography: 'NCEA Guided Search (Geographic Data)',
+  results: 'NCEA Search Results Summary',
+  generalTab: 'NCEA Catalogue Detail (General)',
+  accessTab: 'NCEA Catalogue Detail (Access)',
+  naturalCapitalTab: 'NCEA Catalogue Detail (Natural Capital)',
+  qualityTab: 'NCEA Catalogue Detail (Quality)',
+  geographyTab: 'NCEA Catalogue Detail (Geography)',
+  governanceTab: 'NCEA Catalogue Detail (Governance)',
+  licenseTab: 'NCEA Catalogue Detail (License)',
```

Comment:
it is better to use hyphen than () on the SEO aspects. title should be **NCEA Catalogue Detail - Quality**

Reply:
As per the current AC i am approving this PR. But we need to changes this once its been discussed with the team on refinement